src/main/haskell/kore/test/Test/Kore/Step/Function/UserDefined.hs

```diff
@@ -59,26 +59,7 @@ import           Test.Tasty.HUnit.Extensions
 
 test_userDefinedFunction :: [TestTree]
 test_userDefinedFunction =
-    [ testCase "Cannot apply function if step fails"
```

Comment:
Why did you delete this test? It should have still failed because not all free variables were included in the substitution or something similar.